packages/terra-form/tests/nightwatch/textarea/PopulatedTextarea.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import Textarea from '../../../lib/Textarea';
+
+const textarea = () =>
+  <Textarea
+    name="job_experience"
+    defaultValue=""
+    isInline
```

Comment:
Removed it here. https://github.com/cerner/terra-core/pull/410/commits/a6d5d235c4e1ca12bf42865c62387b91aaa9839f